monitoring/definitions/shared/workerutil.go

```diff
@@ -126,7 +126,9 @@ func (workerutilConstructor) NewGroup(containerName string, owner monitoring.Obs
 	if len(row) == 5 {
 		// If we have all 5 metrics, put handlers on a row by itself first,
 		// followed by the standard observation group panels.
-		rows = []monitoring.Row{row[:1], row[1:]}
+		secondRow := make(monitoring.Row, len(row[1:]))
+		copy(secondRow, row[1:])
+		rows = []monitoring.Row{row[:1], secondRow}
```

Comment:
```suggestion
		firstRow :=[]Observable{row[0]}
		rows = []monitoring.Row{firstRow, secondRow}
```

To be even more specific